backend-service/src/main/java/com/smartparkinglot/backendservice/service/driverservice/DriverServiceImpl.java

```diff
@@ -44,9 +44,11 @@ public Driver addOrSave(Driver driver) {
         if (driver_id == null && driverRepository.findByEmail(driver_email) != null ){
             Driver found_driver = driverRepository.findByEmail(driver_email);
             if (found_driver.getIsDeleted()){
+                /*
                 found_driver.setIsDeleted(false);
                 driverRepository.save(found_driver);
-                throw new AccountActivatedException("Account reactivated again.");
+                 */
```

Comment:
Why did you leave a comment here? What's the point?

Reply:
There is no real reason for that. Just want to keep it there, dont mind please.